scripts/start.py

```diff
@@ -105,6 +109,9 @@ def main(args=None):
         '' if parsed_args.save_datastore else '--clear_datastore=true')
     enable_console_arg = (
         '--enable_console=true' if parsed_args.enable_console else '')
+    disable_host_checking_arg = (
+        '--enable_host_checking=False'
```

Comment:
`False` or `false`? (above U see `true`, so just wanted to check!)

Reply:
Both seem to work, renamed to 'false'